stats/stream

```diff
@@ -3,8 +3,10 @@
 LKP_SRC = ENV['LKP_SRC'] || File.dirname(File.dirname(File.realpath($PROGRAM_NAME)))
 
 require "#{LKP_SRC}/lib/log"
+require "#{LKP_SRC}/lib/statistics"
 
 stats = {}
+stats.default_proc = proc { |hash, key| hash[key] = [] }
```

Comment:
above 2 lines can be combined like stats = Hash.new { |h, k| h[k] = [] }

Reply:
The change is done according to the review comment.